content/asp.net/getting-started/mvc.md

```diff
@@ -1,20 +1,122 @@
 # Introducing ASP.NET Core MVC
-by [Steve Smith](http://deviq.com/me/steve-smith)
+by [James Chambers](http://jameschambers.com)
 
 #### Sample Files
 Download a ZIP containing this tutorial's sample files:
-- [Initial Version] - Use this as a starting point when following along with the tutorial yourself
-- [Completed Version] - Includes the completed versions of all samples
+- [Completed Version (Pending)] - Includes the completed versions of all samples
 
-See the [Issue](https://github.com/dotnet/training-tutorials/issues/59) to claim this lesson and/or view what should be included in it.
+If you don't already have a copy of Visual Studio 2015, you can download a free copy here:
+- [Visual Studio 2015 Community Edition](https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx)
 
-## First Header
+Please ensure you have the latest updates:
+- [Update Visual Studio 2015](https://www.visualstudio.com/downloads/download-visual-studio-vs)  
 
-Start the lesson here.
+>MVC Core builds on the [ASP.NET Core](toc.md) essentials and gives us a framework which we can use to process incoming requests with a structured approach. We use default conventions to take advantage of things like routing, dependency injection and view rendering.
+>
+>The remainder of this tutorial will use the sample files above as a reference. You can follow along with these next lessons to build the sample on your own, or download the full project and run the completed solution locally.
+>
+>We will use Visual Studio 2015 for these exercises. Any version will work provided you are running the latest version.
 
-...
+## Creating Your Project 
+Open up Visual Studio 2015 and create a new project from the File -> New -> Project menu. Under the templates selection, choose Web, then pick ASP.NET Core Web Application (.NET Core) from the list of available templates. Name your project "MvcBasics".
 
+![Select your project template](images\visual-studio-project-template.png)
```

Comment:
I think you need to use `/` in the path, instead of `\` (at least to make it show on GitHub, not sure about the actual site).